inst/schema/NestedFilterOption.schema.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Most filter options can be simple string arrays, but this allows for options which contain further options